src/panel/spoiler_card/SpoilerCardContainer.js

```diff
@@ -64,6 +65,7 @@ class SpoilerCardContainer extends React.Component {
 
 	handleSaveTags() {
 		spoilerCardActions.editTags(this.props.index, this.title, this.tags);
+		this.tags = this.tags.trim();
 	}
```

Comment:
Nit:
Tags should be trimmed individually (after comma separation) anyways, so trimming here may be unnecessary.